modules/11_aerosols/exoGAINS2025/datainput.gms

```diff
@@ -0,0 +1,300 @@
+*** |  (C) 2006-2024 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/11_aerosols/exoGAINS2025/datainput.gms
+
+*** AP scenario and SSP selection -----------------------------------------------------------------------
+*** if cm_APssp is set to FROMGDPSSP, set it to the value of cm_GDPpopScen
+$ifthen.derivessp1 "%cm_APssp%" == "FROMGDPSSP"
+$setGlobal cm_APssp  "%cm_GDPpopScen%"
+$endif.derivessp1
+*** if cm_APscen is set to MTFR or SMIPVLLO, set cm_APssp accordingly
+$ifthen.derivessp2 "%cm_APscen%" == "MTFR"
+$setGlobal cm_APssp  "MTFR"
+$elseIf.derivessp2 "%cm_APscen%" == "SMIPVLLO"
+$setGlobal cm_APssp  "SMIPVLLO"
+$endif.derivessp2
+*** SSP4 not available for SMIPbySSP scenario
+$ifthen.checkscen1 "%cm_APscen%" == "SMIPbySSP"
+$ifthen.checkssp1  "%cm_APssp%" == "SSP4"
+abort "SSP4 not available for SMIPbySSP scenario. Please select another scenario x ssp combination."
+$endif.checkssp1
+$endif.checkscen1
+*** GAINSlegacy not available in exoGAINS2025
+$ifthen.checkssp2  "%cm_APssp%" == "GAINSlegacy"
+abort "GAINSlegacy not supported by exoGAINS2025. Please switch to exoGAINS for using GAINSlegacy.";
+$endif.checkssp2
+
+*** initialize p11_share_trans with the global value, will be updated after each negishi/nash iteration
+p11_share_trans("2005",regi) = 0.617;
+p11_share_trans("2010",regi) = 0.625;
+p11_share_trans("2015",regi) = 0.626;
+p11_share_trans("2020",regi) = 0.642;
+p11_share_trans("2025",regi) = 0.684;
+p11_share_trans("2030",regi) = 0.710;
+p11_share_trans("2035",regi) = 0.727;
+p11_share_trans("2040",regi) = 0.735;
+p11_share_trans("2045",regi) = 0.735;
+p11_share_trans("2050",regi) = 0.742;
+p11_share_trans("2055",regi) = 0.736;
+p11_share_trans("2060",regi) = 0.751;
+p11_share_trans("2070",regi) = 0.774;
+p11_share_trans("2080",regi) = 0.829;
+p11_share_trans("2090",regi) = 0.810;
+p11_share_trans("2100",regi) = 0.829;
+p11_share_trans("2110",regi) = 0.818;
+p11_share_trans("2130",regi) = 0.865;
+p11_share_trans("2150",regi) = 0.872;
+
+*** GAINS2025 emission factors --------------------------------------------------------------------------
+parameter f11_emifacs_sectREMIND_sourceCEDS(tall,all_regi,all_enty,all_enty,all_te,all_sectorEmi,emisForEmiFac,all_APscen,all_APssp)     "GAINS2025 emission factors weighted by CEDS emissions"
+/
+$ondelim
+$include "./modules/11_aerosols/exoGAINS2025/input/f11_emifacs_sectREMIND_sourceCEDS.cs4r"
```

Comment:
if this realization should be default, you need to provide input data for this, i.e. at least rev 7.65